docs/resources/server.md

```diff
@@ -7,43 +7,46 @@ description: |-
   ~> This resource is in beta and may be subject to breaking changes in the future. Use with caution. See our guide https://registry.terraform.io/providers/stackitcloud/stackit/latest/docs/guides/opting_into_beta_resources for how to opt-in to use beta resources.
   Example Usage
   With key pair
-  ```terraform
-  resource "stackitkeypair" "keypair" {
+  
```

Comment:
unrelated changes, not sure if they were missed in a previous PR or if you didn't merge with main

Reply:
As discussed, unrelated change caused by missing doc generation on main, but will be kept to cleanup main as well.